requirements/requirements.txt

```diff
@@ -1,3 +1,14 @@
+# version fix
```

Comment:
Can these dependency versions be looser? Locked versions might cause conflicts wither other plugins/skills installed to the same venv

Reply:
Agree. This version lock appeared after agreement of tensorflow and neon versions(as tensorflow has a wide range of deps)
Deps are locked very strong(some package locks it in upper range, some in bottom range)
And it is more about order of installs. tensorflow installed highest compatible python-dateutil==2.7.0 but for lingua-franca 2.6.0 is needed
1.15.0 <= six < 1.16.0 (tensorflow)
5.4 <= PyYAML <6.0(ovos-utils)
python-dateutil == 2.6.0 (lingua-franca)

In future it can be reduced by full removement of TensorFlowTTS, as it has a lot of deps for NN training
But when we will convert models to tflite, there will be only numpy requirement

Reply:
Got it. This should be okay then for this version. I think sometime soon we should work out a constraints file we can use at install time so each package can specify loose dependencies so we don't get artificial incompatibility.

Something else I noticed in workflows is `pip install .` appears to do a better job of bactracking dependencies than `pip install requirements.txt` (https://github.com/NeonGeckoCom/neon-skill-utils/pull/273/commits/0dfa551f07cb3dab879b8201da6fcf78fd6d9918) though I never found documentation explaining why